src/converter.ts

```diff
@@ -183,7 +183,7 @@ export default async function (ast: ParseResult<File>, targetedDiscordModApiLibr
         }
     }
     parsedBodyWithoutOurImports.unshift(...await addCode(targetedDiscordModApiLibrary.default));
-    if ((targetedDiscordModApiLibrary as { default: IModImplementation } & { convertFormat: (ast_: Statement[]) => Statement[] }).convertFormat == undefined)
+    if (shouldConvertFormat == false || (targetedDiscordModApiLibrary as { default: IModImplementation } & { convertFormat: (ast_: Statement[]) => Statement[] }).convertFormat == undefined)
```

Comment:
Actually, could you use `!shouldConvertFormat` ?

Reply:
what if it is a string?

Reply:
networking taught me to never trust the user

Reply:
`shouldConvertFormat` is a boolean. What "string" are you referencing?

Reply:
what if somehow user passes a string to that function

Reply:
`!!`